src/command_modules/azure-cli-acr/README.rst

```diff
@@ -84,23 +84,36 @@ Update a container registry
 
     Command
         az acr update: Update a container registry.
-
+    
     Arguments
-        --name -n      [Required]: Name of container registry.
-        --disable-admin          : Disable admin user.
-        --enable-admin           : Enable admin user.
-        --resource-group -g      : Name of resource group.
-        --storage-account-name -s: Name of an existing storage account.
```

Comment:
If you want to, you can keep the convenience flags you already have as they are arguably (IMO anyways) a better user experience than the generic updates. `az network nic update` is an example of how to do this with minimal modification to your existing custom command.

If you prefer to just expose the generic update, that's fine too.